app/assets/stylesheets/font-awesome.min.css

```diff
@@ -1,4 +1,2198 @@
 /*!
```

Comment:
@GauravMakwana-BTC : Can we use gem 'font-awesome-rails' instead of css & import that in our application.css